src/test/unittest/unittest.sh

```diff
@@ -2287,7 +2289,57 @@ function create_holey_file_on_node() {
 #
 # setup -- print message that test setup is commencing
 #
+# arguments:
+#  -t test-type
+#  -f fs-type
+#  -b build-type
+#  -v tool=req - configures Valgrind
+#  -c - executes "check" function at the end
```

Comment:
Hm, what about set check as "default" and then add something like -c skip option?
Probably there is much more tests with check than without check.(?) - if not then this solution is ok

Reply:
Indeed. 70% of tests use "check" function.

Reply:
Done